packages/flutter/test/flutter_test_config.dart

```diff
@@ -27,9 +27,9 @@ Future<void> testExecutable(FutureOr<void> Function() testMain) {
 
   // TODO(polina-c): clean up leaks and stop ignoring them.
   // https://github.com/flutter/flutter/issues/137311
+  // Leak tracking is off by default. To temporary enable it add `.withTrackedAll()` after `settings`.
   LeakTesting.settings = LeakTesting
     .settings
```

Comment:
This is fine to unblock for now but you might want to consider using an environment variable to allow users to turn on leak tracking by setting the environment variable when calling `flutter test`.

Reply:
Is there difference in effort to add flag and to add this line of code?

Created PR that will show memory leaks: https://github.com/flutter/flutter/pull/138581
The plan is to refresh it manually and check if there are leaks.

Do you happen to have other automation in mind?